options_block_based_table.go

```diff
@@ -235,3 +235,9 @@ func (opts *BlockBasedTableOptions) SetFormatVersion(version int) {
 func (opts *BlockBasedTableOptions) SetIndexType(value IndexType) {
 	C.rocksdb_block_based_options_set_index_type(opts.c, C.int(value))
 }
+
+// SetFlushEveryBlockPolicy sets the table to perform a flush after inserting each key.  Useful for
```

Comment:
Can we not add this function in this PR if it's only for testing? In tests you can call `Flush` if you need.